exercises/Mathieu_Tillet/ex02_oo_basics/src/shapesorter.cpp

```diff
@@ -0,0 +1,47 @@
+#include "shapesorter.h"
+
+ShapeSorter::ShapeSorter(){}
```

Comment:
General remarks for the methods here: for the names we use lower case camel case, so `printBySides`, then for the arguments that are const, state those explicitly so in the signature, for printing you can use the range based loop, and normally we also require doxygen comments as developer documentation.